web/src/Routes.tsx

```diff
@@ -25,12 +25,18 @@ const CustomerExplorePage = lazy(() => import('components/customer/explore'));
 // Merchant Pages
 const MerchantLandingPage = lazy(() => import('components/merchant/landing'));
 const MerchantSignUpPage = lazy(() => import('components/merchant/sign-up'));
+// Shared Pages
+const ListingDetailsPage = lazy(() => import('components/listing-details'));
 // Design samples
 const DesignSamplesPage = lazy(() => import('components/design-samples'));
 
 const Routes: React.FC = () => (
   <Suspense fallback={<Loading />}>
     <Switch>
+      <CustomerCommitCountProvider>
```

Comment:
we already have a <CustomerCommitCountProvider> below

Reply:
@ifhuang Where? It is the closing tag in line 39?

Reply:
line 43

Reply:
😅 Oh just saw it. My bad for not realising that when dealing with all the merge conflicts earlier on! 😭